doc/rst/technotes/gpu.rst

```diff
@@ -428,6 +446,20 @@ file containing the outlined loop and ``num`` as the line number of the loop
 header. For example, a kernel on line 3 of ``chpl.foo`` will be named
 ``chpl_gpu_kernel_foo_line_3_``).
 
+Oversubscribing GPUs With Chapel Tasks
```

Comment:
Not sure how I feel about using the term "Oversubscribing" here.

I guess I'm imprinted to think of "oversubscribing" as being a term we use for having multiple locales for gasnet configurations on the same node. IOW I was expecting to read about how we could set `here.gpus[0]` and `here.gpus[1]` to refer to the same physical HW (which would be a cool feature BTW, just not what you're talking about here).

I think I'd just call what you have here "Multitasking on the GPU" though maybe there's some better term.

Reply:
How about we keep it vague "Chapel Tasks and GPU Execution"?

Reply:
> How about we keep it vague "Chapel Tasks and GPU Execution"?

I'm good with that.